### PR TITLE
Document traffic_ctl monitor command

### PR DESCRIPTION
**Pull request by cmcfarlen**

### CODE REVIEW

**Comment by maskit** [comment]
Maybe `whats-new.en.rst` is more appropriate.

File: doc/release-notes/whats-new.en.rst [hunk 1]
```diff
@@ -31,6 +31,8 @@ total of <z> contributors have participated in this development cycle.
 New Features
 ------------
 
+- traffic_ctl has a new command ``monitor`` do show a continuously updating list of
```
Comment (JosiahWI) [inline doc/release-notes/whats-new.en.rst:34]:
```suggestion
- traffic_ctl has a new command ``monitor`` to show a continuously updating list of
```

**Review by JosiahWI — CHANGES_REQUESTED** [review]

**Review by brbzull0 — DISMISSED** [review]

**Review by brbzull0 — DISMISSED** [review]

File: doc/release-notes/whats-new.en.rst (hunk 1 above)
Comment (JosiahWI) [reply doc/release-notes/whats-new.en.rst]: This hasn't changed. Is the original version correct?

Comment (cmcfarlen) [reply doc/release-notes/whats-new.en.rst]: I tried to use GitHub's suggestion feature but it seems it didn't actually do the change.  I'll try manually.

**Review by JosiahWI — APPROVED** [review]

**Review by maskit — APPROVED** [review]

**Pull request merged**